Framework/WorkflowAlgorithms/inc/MantidWorkflowAlgorithms/MuonGroupAsymmetryCalculator.h

```diff
@@ -35,16 +35,19 @@ class DLLExport MuonGroupAsymmetryCalculator : public MuonGroupCalculator {
   MuonGroupAsymmetryCalculator(const API::WorkspaceGroup_sptr inputWS,
                                const std::vector<int> summedPeriods,
                                const std::vector<int> subtractedPeriods,
-                               const int groupIndex);
+                               const int groupIndex, const double start = 0.0,
+                               const double end = 30.0);
   /// Performs group asymmetry calculation
   API::MatrixWorkspace_sptr calculate() const override;
 
 private:
   /// Removes exponential decay from the workspace
   API::MatrixWorkspace_sptr removeExpDecay(const API::Workspace_sptr &inputWS,
                                            const int index) const;
+  API::MatrixWorkspace_sptr
+  EstimateAsymmetry(const API::Workspace_sptr &inputWS, const int index) const;
 };
-
+std::vector<double> convertToVec(std::string const &list);
```

Comment:
Function already available in https://github.com/mantidproject/mantid/blob/master/Framework/Kernel/src/VectorHelper.cpp#L442, consider reusing.

Also in https://github.com/mantidproject/mantid/pull/19083/commits/ac8ee01662790218203650e58dc8e04d8094f004#diff-75c8ef42f54a40b0f753f65890821da9R77